src/main/java/com/google/api/codegen/config/DiscoveryField.java

```diff
@@ -72,6 +73,13 @@ public String getFullName() {
   }
 
   @Override
+  public TypeName getTypeName(ImportTypeTable typeTable) {
+    return typeTable
+        .getTypeTable()
+        .getTypeName(((SchemaTypeTable) typeTable).getFullNameFor(schema));
```

Comment:
This has an odd look - I see "type table" referenced three times in this expression..